### PR TITLE
feat(person-listitem): plain property

### PR DESCRIPTION
**Pull request by AndrejNikolicEq**
# Person
## Person ListItem

**ListItem - new feature**: `outlined` property added for adding border and border-radius from list item component
**Note:** Fixes problem with using list item as selected person in PersonSelect component

### CODE REVIEW

**Review by odinr — CHANGES_REQUESTED** [review]
less is more

the frame is just from re-factor of previous code, the frame should not be here, the parent should add this frame _if required_

just make the version major and remove border

btw, classes in WC do not require BEM, since their styling and classes are contained within the shadow DOM. _the bem impl. is just a misunderstanding from previous dev, since the base classes from material design (which uses the plain js/css from root)_

**Comment by AndrejNikolicEq** [comment]
@odinr Was thinking of quick fix, until we get the new design.
But, since we are doing this, gonna swap whole css with styled-components and tokens from EDS

File: .changeset/odd-camels-know.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+'@equinor/fusion-wc-person': minor
```
Comment (eikeland) [inline .changeset/odd-camels-know.md:2]: Make this a major version bump.

**Comment by asbjornhaland** [comment]
What is the status of this? should it still be open?

**Comment by AndrejNikolicEq** [comment]
> What is the status of this? should it still be open?

We can leave it open for now, need to check if we still want it

**Comment by AndrejNikolicEq** [comment]
We do not need it any more, we have new component that covers it

**Pull request closed without merging**